calcom/cal.com
# Fixed Google Calendar custom destination calendar deletion

## What does this PR do?

Added the event object as an parameter of the delete function, so the correct destination calendar is used.

Fixes #1481

## Type of change


## How should this be tested?

1. See steps to reproduce in #1481
2. Delete the booking and it should not throw an error
3. The event should be removed from the Google calendar

## Checklist:
